.github/workflows/acceptance.yml

```
@@ -6,8 +6,24 @@ on:
   pull_request:
 
 jobs:
+    parse-commit-message:
+      if: ${{ github.ref != 'refs/heads/master' }}
+      runs-on: ubuntu-16.04
+      outputs:
+        commit: ${{ steps.commit.outputs.message }}
+      steps:
+        - uses: actions/checkout@v2
+          with:
+            ref: ${{ github.event.pull_request.head.sha }}
+
+        - name: Parse commit message
+          id: commit
+          run: |
+            echo "::set-output name=message::$(git log --format=%B ${{ github.event.after }})"
```

Shouldn't this be `git log -n1`? And I couldn't find documentation on `github.event.after`, could you link that? Anyway, `show` defaults to `HEAD` so you might not even need that, and is preferable + faster compared to `log` for this purpose.

I reviewed `GIT-SHOW(1)`, `git show -s --format=%B` is what we want here.

```suggestion
            echo "::set-output name=message::$(git show -s --format=%B)"
```

---

Ah makes sense, for pull request events we have a before/after sha where before is first commit in PR and after is last, unfortunately there doesn't seem to be a way to get only the commits in a push for a PR event.